test/morgan.js

```diff
@@ -27,6 +27,22 @@ describe('morgan()', function () {
       .expect(200, cb)
     })
 
+    it('should throw warnings for unspecified format', function (done) {
```

Comment:
I'll take a look into this test to see how I can help, but this test doesn't actually do anything that is being asserted: for example it's not asserting anywhere what the format was changed to `'default'` and it doesn't assert that any warning was actually displayed.